resources/multisite_drupal_standard.make

```diff
@@ -349,9 +349,6 @@ projects[media][patch][] = https://www.drupal.org/files/issues/media-wysiwyg-ove
 ; https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-3650
 projects[media][patch][] = http://www.drupal.org/files/issues/file_entity-remove-contextual-links-2401811-11.patch
 
-projects[media_crop][subdir] = "contrib"
```

Comment:
@gervasek Thanks for reviewing it. I think I miss a point. When is the make file being triggered? According to my understanding the make file is being triggered only for new websites. Could you help me understand more what to do next or try further handling yourself in your best way?

Same with Language Tools features: https://webgate.ec.europa.eu/CITnet/jira/browse/MULTISITE-5739, a redundant feature that we agreed with Wesley it should be removed. It also creates a problem when activated by overriding the whole content translation UI with its own.